public/js/services/presence.js

```diff
@@ -4,7 +4,9 @@ import {logError} from '../util/logger';
 export const subscribeToPresence = (atomType, atomId) => {
   const store = getStore();
   const presenceClient = store.getState().presenceClient;
-  if (Object.keys(presenceClient).length !== 0) {
+  if (Object.keys(presenceClient).length !== 0
+      && presenceClient !== "undefined") {
+    
```

Comment:
This change I have added to reduce errors in DEV environments - `presenceClient` is undefined in local because it fails to connect to workflow statuses (I think). This throws a lot of errors that clutter up the console.

Reply:
minor: an alternative to doing this check twice would be to conditionally render the component based on config. For example, this is how MAM does it - https://github.com/guardian/media-atom-maker/blob/master/public/video-ui/src/components/Header.js#L112-L120

Reply:
@akash1810 - thanks for raising, will look into doing this